ebook/1_latex2html.py

```diff
@@ -0,0 +1,737 @@
+# by Torben Menke https://entorb.net
+
+"""
+run from within ebook dir via 
+python3 1_latex2html.py
+"""
+
+import os
+import re
+import glob
+from datetime import date
+
+# Notes
+# footnotes are converted to inline text
+# these intro parts are skipped:
+#  Based on the characters of J. K. ROWLING
+#  CONTENT WARNINGS
+# Omake chapters are included inplace, not at appendix
+
+# TODO:
+# modify Preamble
+
+today = date.today()
+
+dir_tex_source = "../chapters/"
+dir_tmp = "tmp"
+dir_out = "output"
+for dir in (dir_tmp, dir_out):
+    os.makedirs(dir, exist_ok=True)
+
+
+html_preamble = f"""
+<h2>Preamble of this eBook</h2>
+<p>The original version of this great book 'Harry Potter and the Methods of Rationality' by Eliezer Yudkowsky is:<br/>
+<a href="https://www.hpmor.com">https://www.hpmor.com</a></p>
+<p>The typesetting as well as the reviewed text are adopted from:<br/>
```

Comment:
"revised" rather than "reviewed"

Reply:
thanks.

By the way, shall we publish the resulting epub next to the pdf file? If so, we should add this info here.

Reply:
Yes, please add this info. Sorry I missed this comment last night!

Reply:
what would be the url?

Reply:
currently I generate the resulting .epub file etc. in ebook/output/

Reply:
The `GNUmakefile` will need updating to upload the ebook files as part of a release. The URL for 1.1.3's PDF is https://github.com/rjl20/hpmor/releases/download/v1.1.3/hpmor-1.1.3.pdf so the URLs for other files will be similar, just change the file name mutatis mutandis.

Reply:
(I will take care of `GNUmakefile` if you tell me exactly which files need uploading.)

Reply:
currently the output is
```
ebook/output/hpmor.epub
ebook/output/hpmor.mobi
```
see 2_html2epub.sh lines 40 and 43

concerning the URL: it would be very nice, if we would find a way to point to "latest" release, and include this info in the preamble. Just found out, that this works:
https://github.com/rjl20/hpmor/releases/latest/ :-)

So I modified the preamble text accordingly.

Reply:
Good stuff! Please can you allow me to edit the PR, so I can add the uploading of ebooks to releases? https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
That flag was already set, please try editing

Reply:
Thanks, unfortunately according to my research there's no way for me to find out whether the flag is set (other than by trying an edit). I'll get on to it as soon as I can find a moment.